versioned_docs/version-1.2/quickstarts/ratify-on-azure.md

```diff
@@ -449,13 +449,13 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
     apiVersion: config.ratify.deislabs.io/v1beta1
     kind: KeyManagementProvider
     metadata:
-    name: keymanagementprovider-akv
+      name: keymanagementprovider-akv
```

Comment:
This needs to be reflected in the following kubectl commands as well:
KeyManagementProvider-akv --> keymanagementprovider-akv